olive/model/__init__.py

```diff
@@ -551,7 +551,21 @@ def load_mlflow_model(self):
 
         with open(os.path.join(self.model_path, "MLmodel"), "r") as fp:
             mlflow_data = yaml.safe_load(fp)
-            hf_pretrained_class = mlflow_data["flavors"]["hftransformers"]["hf_pretrained_class"]
+            # default flavor is "hftransformersv2" from azureml.evaluate.mlflow>=0.0.8
+            # "hftransformers" from azureml.evaluate.mlflow<0.0.8
+            # TODO: let user specify flavor name if needed
+            hf_pretrained_class = None
+            flavors_names = ["hftransformersv2", "hftransformers"]
+            for flavors_name in flavors_names:
+                if flavors_name in mlflow_data.get("flavors", {}):
+                    hf_pretrained_class = mlflow_data["flavors"].get(flavors_name, {}).get("hf_pretrained_class", None)
+                    break
+            if not hf_pretrained_class:
+                raise ValueError(
```

Comment:
```python
            for flavors_name in flavors_names:
                flavors_name_config = mlflow_data["flavors"].get(flavors_name)
                if flavors_name_config:
                    hf_pretrained_class = flavors_name_config["hf_pretrained_class"]
                    break

```

Reply:
'hf_pretrained_class' may not exist.

Reply:
If not exist, it will raise KeyError

Reply:
https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-evaluate-mlflow/azureml/evaluate/mlflow/hftransformers/__init__.py&version=GBmaster&line=788&lineEnd=789&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-evaluate-mlflow/azureml/evaluate/mlflow/hftransformers/__init__.py&version=GBmaster&line=856&lineEnd=858&lineStartColumn=1&lineEndColumn=75&lineStyle=plain&_a=contents

Reply:
https://github.com/mlflow/mlflow/blob/master/mlflow/utils/model_utils.py